### PR TITLE
Enhance gpu multihead matmul v3 fuse pass

### PR DESCRIPTION
**Pull request by wjj19950828**
### PR types
Performance optimization

### PR changes
OPs

### Describe
1、添加两个已有pass到gpu当中：matmul_scale_fuse_pass、multihead_matmul_fuse_pass_v3
2、添加```mul```到对应pattern

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/ir/inference/test_multihead_matmul_fuse_pass_v3.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+import numpy as np
+import paddle.inference as paddle_infer
+from functools import partial
+from typing import Optional, List, Callable, Dict, Any, Set
+import unittest
+
+import hypothesis
+from hypothesis import given, settings, seed, example, assume, reproduce_failure
+import hypothesis.strategies as st
+
```
Comment (heliqi) [inline python/paddle/fluid/tests/unittests/ir/inference/test_multihead_matmul_fuse_pass_v3.py:26]: 后面可以补一个这个pass的图，匹配什么样的子图。可以参考下这个:  https://github.com/PaddlePaddle/Paddle/blob/7bb72e37b21422ca56904b7cc75d2ca832c01327/python/paddle/fluid/tests/unittests/ir/inference/test_fc_elementwise_layernorm_fuse_pass.py#L29

**Review by heliqi — COMMENTED** [review]
LGTM

**Review by Wangzheee — APPROVED** [review]
LGTM

**Review by heliqi — APPROVED** [review]

File: paddle/fluid/framework/ir/multihead_matmul_fuse_pass.cc [hunk 2]
```diff
@@ -424,7 +424,8 @@ PDNode* MultiHeadMatmulPattern::operator()() {
 }
 
 PDNode* MultiHeadMatmulV3Pattern::operator()() {
-  std::unordered_set<std::string> matmul_ops{"matmul", "matmul_v2"};
+  // Add mul op to support huggingface onnx model convertsion by x2paddle
```
Comment (jiangjiajun) [inline paddle/fluid/framework/ir/multihead_matmul_fuse_pass.cc:427]: convertsion -> conversion

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

**Review by jiangjiajun — APPROVED** [review]

**Review by jiangjiajun — APPROVED** [review]

**Pull request merged**